zipline/test_algorithms.py

```diff
@@ -485,6 +485,11 @@ def initialize(self, max_leverage=None):
         self.set_max_leverage(max_leverage=max_leverage)
 
 
+class SetMinLeverageAlgorithm(TradingAlgorithm):
+    def initialize(self, min_leverage=None, deadline=None):
```

Comment:
Should we just make these required if it's invalid to pass `None` for these values to `set_min_leverage`?